src/components/tool/guide/Instructions/StartInstructions.js

```diff
@@ -34,7 +35,7 @@ export function LyraStartInstructions({
             In the ssh session, run the following command to schedule the {jobType.toLowerCase()} job:
         </InstructionText>
         <CopyBox>
-            {`qsub${jobType === "Interactive" ? " -I" : ""} -l walltime=1:00:00 -l ${resources.join(":")}`}
+            {`qsub${jobName !== "" ? " -N \"" + jobName + "\"" : ""}${jobType === "Interactive" ? " -I" : ""} -l walltime=1:00:00 -l ${resources.join(":")}`}
```

Comment:
If the job name is specified when it's a batch job, then later changed to an interactive job, it will be displayed here but won't be able to be changed.

Reply:
Fixed